Tests/SwiftParserTest/RegexLiteralTests.swift

```diff
@@ -751,13 +751,13 @@ final class RegexLiteralTests: XCTestCase {
   }
 
   func testBinOpDisambiguation26() {
-    // FIXME: The diagnostic should be one character back
     assertParse(
       """
-      # 1️⃣/^ x/
+      #1️⃣ 2️⃣/^ x/
       """,
       diagnostics: [
-        DiagnosticSpec(message: "expected identifier in macro expansion", fixIts: ["insert identifier"])
+        DiagnosticSpec(locationMarker: "1️⃣", message: "extraneous whitespace after '#' is not permitted", fixIts: ["remove whitespace"]),
```

Comment:
Bit weird to have both (1) and (2). Ideally we'd just have (2) with the trivia transferred to the trailing of the inserted identifier.

Reply:
Changed it in a separate commit to this PR.